### PR TITLE
add chrome options arguments

### PR DESCRIPTION
**Pull request by suciista09**
user can add chrome option arguments in configuration file

### CODE REVIEW

**Comment by ghost** [comment]
Hi @suciolx 

Could you please open a similar PR against https://github.com/athena-oss/php-fluent-webdriver-client? We have extracted the WebDriver client code to this new repo.

**Comment by pproenca** [comment]
Hey @suciolx , does this PR still make sense?

**Comment by suciista09** [comment]
@pproenca Hi Pedro, we still using this branch to test our api and BDD on mobile browser. API because we need to pass verify option to test https. And for BDD, We need to add some arguments in chrome options. But as I see in latest plugin-php, there's many changes and I haven't try it. 
 I am sorry for late reply :)

File: src/Api/ApiClientBuilder.php [hunk 1]
```diff
@@ -93,6 +93,7 @@ public function build()
         }
 
         $guzzleClient->setDefaultOption('exceptions', $this->httpExceptions);
+        $guzzleClient->setDefaultOption('verify',false);
```
Comment (pproenca) [inline src/Api/ApiClientBuilder.php:96]: This option should be configurable using `athena.json`

Comment (suciista09) [reply src/Api/ApiClientBuilder.php:96]: So as we talked in Slack, it needs changes in code, right?

**Pull request closed without merging**